src/ILedController.h

```diff
@@ -8,7 +8,7 @@
 #define LOG_TAG_LED "ILedController"
 
 // Pattern types supported:
-enum  Pattern { NONE, RAINBOW_CYCLE, THEATER_CHASE, COLOR_WIPE, CYLON, FADE, RESCUE_FLASH_LIGHT, PULSE, SLIDE, BATTERY_INDICATOR};
+enum  Pattern { NONE, RAINBOW_CYCLE, TRANS_PRIDE, THEATER_CHASE, COLOR_WIPE, CYLON, FADE, RESCUE_FLASH_LIGHT, PULSE, SLIDE, BATTERY_INDICATOR};
```

Comment:
Why did you add it at this position. This will change the order. Wouldn't it be better to add new pattern at the end of the enum?

Reply:
Absolutely. Added fix.